### PR TITLE
Clean up view and convert more methods

### PR DESCRIPTION
**Pull request by pentacular**

### CODE REVIEW

File: api/shape/sort.js [hunk 1]
```diff
@@ -7,65 +7,65 @@ const X = 0;
...
+export const sort = Shape.registerMethod2(
   'sort',
-  (spec = 'z<y<x<') =>
-    async (shape) => {
-      let leafs = [];
-      for (const leaf of getLeafs(await shape.toGeometry())) {
-        const [min, max] = measureBoundingBox(leaf);
-        const shape = await Shape.fromGeometry(leaf);
-        leafs.push({ min, max, shape });
+  ['inputGeometry', 'string'],
```
Comment (BarbourSmith) [inline api/shape/sort.js:12]: Are you converting all of these by hand or is there some sort of macro to do it?

**Review by BarbourSmith — APPROVED** [review]
I'm not sure if these missing diffs are indicative of some sort of issue (probably not, probably just github), but other than that looks great to me 👍 👍 

image

File: api/shape/sort.js (hunk 1 above)
Comment (pentacular) [reply api/shape/sort.js:12]: Manually, for my sins. :)

Hopefully it will get simpler over time.

Comment (BarbourSmith) [reply api/shape/sort.js:12]: 😬 Good luck!

**Pull request merged**